website/api/controllers/webhooks/receive-usage-analytics.js

```diff
@@ -19,12 +28,32 @@ module.exports = {
   },
 
 
-  fn: async function ({anonymousIdentifier, fleetVersion, numHostsEnrolled}) {
+  fn: async function ({
+    anonymousIdentifier,
+    fleetVersion,
+    licenseTier,
+    numHostsEnrolled,
+    numUsers,
+    numTeams,
+    numPolicies,
+    softwareInventoryEnabled,
+    vulnDetectionEnabled,
+    systemUsersEnabled,
+    hostStatusWebhookEnabled,
+  }) {
 
     await HistoricalUsageSnapshot.create({
       anonymousIdentifier,
       fleetVersion,
-      numHostsEnrolled
+      licenseTier,
+      numHostsEnrolled,
+      numUsers,
+      numTeams,
+      numPolicies,
+      softwareInventoryEnabled,
+      vulnDetectionEnabled,
+      systemUsersEnabled,
+      hostStatusWebhookEnabled,
     });
```

Comment:
This is good and explicit, no complaints.

FYI, another option when it's a straight up pass-thru like this:

```js
fn: async function(inputs) {
  await HistoricalUsageSnapshot.create(inputs);
}
```